dbt_platform_helper/COMMANDS.md

```diff
@@ -1016,22 +1017,62 @@ platform-helper database dump --account-id <account_id> --app <application>
 ## Usage
 
 ```
-platform-helper database load --account-id <account_id> --app <application> 
-                              --env <environment> --database <database> 
-                              --vpc-name <vpc_name> 
+platform-helper database load --to <to_env> --database <database> 
+                              [--app <application>] [--to-vpc <to_vpc>] 
+                              [--auto-approve] 
 ```
 
 ## Options
 
-- `--account-id <text>`
-
 - `--app <text>`
+  - The application name. Required unless you are running the command from your deploy repo
+- `--to <text>`
+  - The environment you are loading data into
```

Comment:
Are these mandatory parameters or are they only required if not in deploy repo?  (dump command above would suggest the latter)